assignment6/rearranging_cars_test.py

```diff
@@ -0,0 +1,26 @@
+'''
+Created on 07 July 2017
+
+@author: Eva
+'''
+
+import unittest
+from rearranging_cars import rearrange_cars, Move
+
+class UnknownLanguageTest(unittest.TestCase):
+    def setUp(self):
```

Comment:
When you will have more tests, using `setUp` becomes unreasonable. Instead of it, in `testKnownCase` you can write smth like `self.assertTrue(self.check_list_contents(rearrange_cars([1, 2, 3, 0], [3, 2, 1, 0]), [Move(..., ...), ....]))`
Moreover, `setUp` makes sence if you really need to reuse some values. But here you use them only once.